halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/google/GoogleAddAccountCommand.java

```diff
@@ -65,6 +65,13 @@ protected String getProviderName() {
   )
   private String userDataFile;
 
+  @Parameter(
+      names = "--regions",
+      variableArity = true,
+      description = "A list of regions for caching and mutating calls. This overwrites any default-regions set on the provider."
```

Comment:
Can you do the same for this description? I see you added the add/remove description already